### ISSUE TITLE
Does Instagram enforce any IP ban or CAPTCHA?

### ISSUE THREAD

**Issue by andreyuhai** [rarcega/instagram-scraper#519]
Hey there,

Does instagram enforce any IP ban or CAPTCHA upon multiple requests? 
I would like to know whether anyone using this scraper encountered with any problems while scraping huge loads.

**Comment by 501st-alpha1** [comment]
I've never seen Captchas, or any bans that affect my entire account, but have had a couple other issues.

The main one is I've gotten is some repeated 429 errors after too many downloads (see other open issues for that).

Also once or twice the login started failing, but that was usually because the cookie file had been corrupted, and the login only failed for the script.  Switching to a new IP was enough to fix that, and eventually this "soft-ban" expired, and I was able to run the script again from the original IP.

**Comment by steffanjensen** [comment]
yes they do i have been ip banned for 14 days once, mostly because i abused the login (by a mistake).. 
This is the only there seem to trigger a real ban.. The instagram was just blank for 14 days on all my deviceses connected to the ip..
